src/components/core/PivotTable.tsx

```diff
@@ -315,34 +314,39 @@ export const RowLoadingElement = (props: ICellRendererParams) => {
 };
 
 export const getDrillIntersection = (
-    drillItems: IDrillItem[],
+    drillItems: IMappingHeader[],
     afm: AFM.IAfm
 ): IDrillEventIntersectionElement[] => {
     // Drilling needs refactoring: all '' should be replaced by null (breaking change)
     // intersection consists of
         // 0..1 measure
         // 0..1 row attribute and row attribute value
         // 0..n column attribute and column attribute values
-    return drillItems.map((drillItem: IDrillItem) => {
-        const { identifier, uri, title } = drillItem;
+    return drillItems.map((drillItem: IMappingHeader) => {
+        let headerLocalIdentifier = null;
+        let headerIdentifier = '';
+        let uriAndIdentifier = null;
+
+        if (!isMappingHeaderAttributeItem(drillItem)) {
+            headerLocalIdentifier = getMappingHeaderLocalIdentifier(drillItem);
+            headerIdentifier = getMappingHeaderIndentifier(drillItem) || '';
+            uriAndIdentifier = headerLocalIdentifier
+                ? getMasterOrDerivedUriAndIdentifier(afm, headerLocalIdentifier)
```

Comment:
could be `getMeasureObjQualifier`?

we also have a type for both uri and identifier
`type ObjQualifier = IObjUriQualifier | IObjIdentifierQualifier;`